django-verdant/rca/templates/rca/base.html

```diff
@@ -116,6 +116,17 @@
             <p class="chromeframe">You are using an <strong>outdated</strong> browser which this site does not fully support. Please <a href="http://browsehappy.com/">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
         <![endif]-->
 
+        <div class="survey-banner" id="#survey-banner">
+            <div class="survey-banner__content">
+                <div class="survey-banner__image-wrapper">
+                    <img class="survey-banner__image" src="{% static "rca/images/amzn100.png" %}" />
```

Comment:
Should have an alt,

```suggestion
                    <img alt="" class="survey-banner__image" src="{% static "rca/images/amzn100.png" %}" />
```

If possible I’d also suggest setting `width` and `height` so the image loading does not move the layout.